docs/reference/ml/df-analytics/apis/get-dfanalytics-stats.asciidoc

```diff
@@ -435,6 +435,15 @@ job is started and memory usage is reported.
 (long)
 The number of bytes used at the highest peak of memory usage.
 
+`status`::::
```

Comment:
Is there an appropriate place in HLRC docs to be updated as well?
I did not find any concrete fields in `docs/java-rest/high-level/ml/get-data-frame-analytics-stats.asciidoc`...

Reply:
Indeed, we don't list the properties of the objects that are returned in the stats response for HLRC. I'd leave it as is for now.